controllers/humiocluster_pods.go

```diff
@@ -696,7 +696,7 @@ func sanitizePod(hc *humiov1alpha1.HumioCluster, pod *corev1.Pod) *corev1.Pod {
 				if envVar.Name == "EXTERNAL_URL" {
 					sanitizedEnvVars = append(sanitizedEnvVars, corev1.EnvVar{
 						Name:  "EXTERNAL_URL",
-						Value: fmt.Sprintf("%s://%s-core-%s.%s:%d", strings.ToLower(string(getProbeScheme(hc))), hc.Name, "", hc.Namespace, humioPort),
+						Value: fmt.Sprintf("%s://%s-core-%s.%s.%s-headless:%d", strings.ToLower(string(getProbeScheme(hc))), hc.Name, "", hc.Name, hc.Namespace, humioPort),
```

Comment:
Wasn't this supposed to say the following to match what we have in the certificate?
```suggestion
						Value: fmt.Sprintf("%s://%s-core-%s.%s-headless.%s:%d", strings.ToLower(string(getProbeScheme(hc))), hc.Name, "", hc.Name, hc.Namespace, humioPort),
```